modules/@apostrophecms/piece-type/ui/apos/components/AposRelationshipEditor.vue

```diff
@@ -27,7 +27,6 @@
       <AposModalRail>
         <AposModalTabs
           v-if="tabs.length"
-          :key="tabKey"
```

Comment:
Useless key since we aren't in a v-for + the props isn't used in `AposModalTabs`.